runelite-client/src/main/java/net/runelite/client/plugins/attackstyles/AttackStylesConfig.java

```diff
@@ -98,24 +98,24 @@ default boolean warnForMagic()
 	}
 
 	@ConfigItem(
-		keyName = "hideAutoRetaliate",
-		name = "Hide auto retaliate",
-		description = "Hide auto retaliate from the combat options tab",
+		keyName = "autoRetaliateHandler",
+		name = "How to handle auto retaliate",
```

Comment:
These names are just a bit too long to fit in the config panel, and could be made less wordy